src/snyk/common/languageServer/settings.ts

```diff
@@ -4,6 +4,7 @@ import { IConfiguration } from '../configuration/configuration';
 export type InitializationOptions = ServerSettings & {
   integrationName?: string;
   integrationVersion?: string;
+  automaticAuthentication?: string;
```

Comment:
should this be a `bool`?

Reply:
Nope, the backend expects a string afaik